### DISCUSSION TITLE
柱状图转换坐标系后如何让标签文字也进行旋转

### DISCUSSION THREAD

**Discussion by juanhuang99** [antvis/G2#6280] [Q&A]
### 问题描述

代码如下：
import { Chart } from '@antv/g2';

const data = [
  { year: '1951 年', sales: 38 },
  { year: '1952 年', sales: 52 },
  { year: '1956 年', sales: 61 },
  { year: '1957 年', sales: 145 },
  { year: '1958 年', sales: 48 },
  { year: '1959 年', sales: 38 },
  { year: '1960 年', sales: 38 },
  { year: '1962 年', sales: 38 },
];

const chart = new Chart({
  container: 'container',
  autoFit: true,
});

chart
  .interval()
  .coordinate({ transform: [{ type: 'transpose' }] })
  .data(data)
  .encode('x', 'year')
  .encode('y', 'sales');

chart.render();

坐标转换之后，如何能让文字也进行旋转，如图所示:
![img_v3_02bj_79c0dfee-c663-4d96-aef4-13d7edc9a38g](https://github.com/antvis/G2/assets/29769760/871f9dc4-682b-42f2-adff-e91d060f3221)

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
试试这样：chart.axis('x', { labelTransform: 'rotate(90deg)' })
